app/test/e2e/login.spec.ts

```diff
@@ -38,13 +38,8 @@ test.describe('Sign in', () => {
 
     // Click sign-in button
     await page.getByTestId('sign-in').click()
-    await page.waitForLoadState('networkidle')
-    // Switch network
-    // Note: default network metamask is Ethereum and the app default network is Sepolia
-    await metamask.approveSwitchNetwork()
 
-    // Wait for connect metamask popup to appear
-    await page.waitForTimeout(3000)
+    await page.waitForLoadState('networkidle')
```

Comment:
Comment this. 

What this is doing ?

I think you should test the case that you are changing the network. 
So you edit the .env parameter and you check the network switch.